src/EditorFeatures/CSharpTest/Outlining/DocumentationCommentOutlinerTests.cs

```diff
@@ -382,5 +382,28 @@ public void TestMultilineSummaryInDocumentationComment2()
 
             AssertRegion(expectedRegion, actualRegion);
         }
+        [Fact, Trait(Traits.Feature, Traits.Features.Outlining)]
+        [WorkItem(2129, "https://github.com/dotnet/roslyn/issues/2129")]
+        public void CrefInSummary()
+        {
+            var tree = ParseLines("/// <summary>",
+                                  "/// Initializes a new instance of the <see cref=\"MyClass\" /> class.",
+                                  "/// </summary>",
+                                  "class C",
+                                  "{",
+                                  "}");
+
+            var typeBlock = tree.DigToFirstTypeDeclaration();
+            var trivia = typeBlock.GetLeadingTrivia();
+
+            var docComment = (DocumentationCommentTriviaSyntax)trivia.Single(t => t.HasStructure).GetStructure();
+            var actualRegion = GetRegion(docComment);
+            var expectedRegion = new OutliningSpan(
+                         TextSpan.FromBounds(0, 98),
+                         "/// <summary> Initializes a new instance of the MyClass class.",
```

Comment:
It is similar to what we showed in VS2013, where we just showed the first line of comment, and gives a hint at where it comes from.